main.py

```diff
@@ -173,7 +201,25 @@ def main():
 		DISPLAY.blit(ballVelLabel, (10, 30))
 		mouseVelLabel = font.render("Mouse velocity: {:.2f}ms-1".format(mouseBall.velocity.mag), 1, BLACK)
 		DISPLAY.blit(mouseVelLabel, (10, 50))
-
+		gravityVelLabel = font.render("Gravity: {}ms-2".format(GRAVITY), 1, BLACK)
+		DISPLAY.blit(gravityVelLabel, (10, 70))
+		diameterVelLabel = font.render("Ball diameter: {:.2f}m".format(ball.radius*2), 1, BLACK)
+		DISPLAY.blit(diameterVelLabel, (10, 90))
+		stringVelLabel = font.render("String length: {:.2f}m".format(string.length), 1, BLACK)
+		DISPLAY.blit(stringVelLabel, (10, 110))
+
+
+		#control prompts
+		stringConnect = font.render("Connect string:           SPACE", 1, BLACK)
+		DISPLAY.blit(stringConnect, (SCREEN_WIDTH-300, 10))
```

Comment:
Problem here is that if the font size changes the text will be incorrectly positioned. Use `stringConnect.get_rect().width` to get the width of the text in pixels, then subtract this from the `SCREEN_WIDTH` - this will position the text with its right edge up against the right edge of the screen, so you can further offset it with minus however far from the right edge you want the text to be.

Same for the other text below.